F1/F1/Constructor/constructorTableViewCell.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  ConstructorTableViewCell.swift
+//  F1
+//
+//  Created by Kaitlyn Sookdhew on 2024/04/08.
+//
+
+import UIKit
+
+class ConstructorTableViewCell: UITableViewCell {
+    
+    @IBOutlet weak private var round: UILabel!
+    @IBOutlet weak private var race: UILabel!
+    @IBOutlet weak private var firstDriver: UILabel!
+    @IBOutlet weak private var firstPoints: UILabel!
+    @IBOutlet weak private var firstPosition: UILabel!
+    @IBOutlet weak private var secondDriver: UILabel!
+    @IBOutlet weak private var secondPoints: UILabel!
+    @IBOutlet weak private var secondPosition: UILabel!
+    
+    static let identifier = Identifiers.constructorTableViewCell
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        firstDriver.layer.cornerRadius = 8
+        firstDriver.layer.masksToBounds = true
+        secondDriver.layer.cornerRadius = 8
+        secondDriver.layer.masksToBounds = true
+    }
+
+    func populateWith(result: Race) {
+        round.text = result.round
+        race.text = result.circuit.location.locality
+        firstDriver.text = result.results.first?.driver.code
+        firstPoints.text = result.results.first?.points
+        firstPosition.text = result.results.first?.position
+
+        if result.results.count > 1 {
```

Comment:
can rather test for if if !result.results.count.isEmpty(). Avoid used count > 1 if you can to make your code look a lot more "professional"

Reply:
Hey, So here im checking that it has at least two not that it is not empty. 😄